content/videos/noc/3-angles/1-angles-and-rotation/index.json

```diff
@@ -5,7 +5,18 @@
   "topics": ["For beginners"],
   "videoId": "DMg-WRfNB60",
   "canContribute": true,
-  "timestamps": [],
+  "timestamps": [
+    { "time": "0:00", "title": "Hello!" },
+    { "time": "0:25", "title": "Explain! Angles" },
```

Comment:
This was an idea we had a while back where we categorized timecodes as "Explain!" or "Code!" etc.  My thinking now is that perhaps this adds too much noise and maybe it makes more sense to keep the timecodes simple, short and sweet when possible. What do you think @kfahn22?

Reply:
I just copied the ones from YouTube.  I agree it is probably unnecessary and  I will edit them.